test/facets/Token/behaviors/bridgeToken.t.sol

```diff
@@ -0,0 +1,198 @@
+    address public tokenAddress;
+
+    function setUp() public virtual override {
+        vm.startPrank(OWNER);
+        super.setUp();
+        initTokenBridge(ARBITRUM_AXELAR_GATEWAY, ARBITRUM_AXELAR_GAS_SERVICE);
+        tokenAddress = address(token);
+        token.addMintingContract(address(token));
+        vm.stopPrank();
+
+        vm.startPrank(MINTER);
+        token.mint(ALICE, ALICE_BALANCE);
+        vm.stopPrank();
+
+        vm.startPrank(ALICE);
+        token.claim();
+        actualAliceBalance = token.balanceOf(ALICE);
+
+        vm.deal(ALICE, 100 ether);
+    }
+
+    /// @notice This function is used to test the onlySupportedChains modifier
+    /// @dev It tests if bridging on an unsupported chain should not be supported
+    function test_bridgingOnUnsupportedChainShouldNotBeSupported() public {
+        vm.expectRevert(TokenBridge__UnsupportedChain.selector);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            10 ether
+        );
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging with zero amount should not be supported
+    function test_bridgingWithZeroAmountShouldNotBeSupported() public {
+        _enableChain(OWNER);
+
+        vm.expectRevert(TokenBridge__NoZeroAmount.selector);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            0
+        );
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging more than the balance should not be supported
+    function test_bridgingMoreThanBalanceShouldNotBeSupported() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+
+        vm.expectRevert(TokenBridge__InsufficientBalance.selector);
+
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            actualAliceBalance + 10
+        );
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging with uint max transfers all the balance
+    function test_bridgingWithUintMaxShouldTransferAllBalance() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            type(uint256).max
+        );
+
+        vm.assertEq(token.balanceOf(ALICE), 0);
+    }
+
+    /// @notice This function is used to test the bridgeToken function
+    /// @dev It tests if bridging with a valid amount should transfer the correct amount
+    function test_bridgingShouldTransferTheCorrectAmount() public {
+        _enableChain(OWNER);
+
+        vm.startPrank(ALICE);
+        ITokenBridge(tokenAddress).bridgeToken{ value: 0.01 ether }(
+            supportedChain,
+            10 ether
```

Comment:
```suggestion
            bridgeAmount
```

And set some variable `uint256 bridgeAmount = 10 ether;`

Reply:
This tests that Alice's balance is decreased; to test that the bridge occurs, more effects must be tested.

Reply:
fixed here https://github.com/giovannifranchi/insrt-v2-contracts/commit/3356b711cd5daca0dacade45bf5aa702964a995c